protokube/cmd/protokube/main.go

```diff
@@ -312,23 +282,23 @@ func run() error {
 	}
 
 	k := &protokube.KubeBoot{
-		Master:            master,
```

Comment:
Would certainly be happy to see gossip dns split out (or protokube split out into an etcd piece, an addons piece and a gossip DNS piece), but my guess is that it isn't the most important thing on your plate :-)  Protokube etcd also sets some gossip DNS hostnames, so it's not totally trivial, though I don't think it's too bad...